python-sdk/tests/prompts/test_prompt_tags.py

```
@@ -1371,3 +1371,65 @@ def test_langwatch_prompts_has_tags_attribute(self, clean_langwatch):
+# PromptsFacade.create() / update() -- parameters passthrough
+# ---------------------------------------------------------------------------
+
+
+class TestPromptsFacadeParameters:
+    """Tests for PromptsFacade forwarding runtime parameters to the API service."""
+
+    def _make_facade_with_mock_api(self) -> PromptsFacade:
+        mock_client = Mock()
+        return PromptsFacade(mock_client)
+
+    def _mock_prompt_data(self, parameters):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_mock_prompt_data`

(ANN202)